01-node-tutorial/answers/app.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http')
+
+const server = http.createServer((req, res)=> {
+    if(req.url === '/') {
+res.end('welcome to our home page')
+    }
+if(req.url ==='/about'){
+    res.end('Here is our short history')
+}
+res.end(`
+<h1>Oops!</h1>
+<p>We can't seem to find the page you are looking for</p>
+<a href="/"> back home</a>
+`)
+})
+
+server.listen(9000)
+
+
```

Comment:
This looks good, just make sure to move this content to a file named `12-http.js` as we will keep adding new modules in the next week's lesson.